variables.tf

```diff
@@ -6,7 +6,7 @@ variable "aws_region" {
 variable "arn_format" {
   type        = string
   default     = "arn:aws"
-  description = "ARN format to be used. May be changed to support deployment in GovCloud/China regions."
+  description = "Deprecated! Calculated automatically by the module. ARN format to be used. May be changed to support deployment in GovCloud/China regions."
```

Comment:
But this is now a major change, isn't it? I tried to avoid it, so everybody can simply use the new module without any change. But I am also fine with it.

Reply:
suggestion added